pom.xml

```diff
@@ -1,301 +1,297 @@
-                        <rabbit.username>myuser</rabbit.username>
-                        <rabbit.password>myuser</rabbit.password>
-                        <rabbit.exchange>ei-exchange</rabbit.exchange>
-                        <rabbit.key>#</rabbit.key>
-                    </systemPropertyVariables>
-                </configuration>
-                <executions>
-                    <execution>
-                        <id>integration-test</id>
-                        <goals>
-                            <goal>integration-test</goal>
-                            <goal>verify</goal>
-                        </goals>
-                        <phase>integration-test</phase>
-                    </execution>
-                </executions>
-            </plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>2.22.0</version>
+				<configuration>
+					<skipTests>${skipUTs}</skipTests>
+					<forkCountTests>${forkCountTests}</forkCountTests>
+					<reuseForks>false</reuseForks>
+					<useSystemClassLoader>false</useSystemClassLoader>
+					<excludes>
+						<exclude>**/*IT.java</exclude>
+						<exclude>${someModule.test.excludes}</exclude>
+					</excludes>
+					<includes>
+						<include>${someModule.test.includes}</include>
+					</includes>
+				</configuration>
+			</plugin>
 
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-site-plugin</artifactId>
-                <version>3.7.1</version>
-                <configuration>
-                    <!-- This is where site.xml is located -->
-                    <siteDirectory>${basedir}/wiki</siteDirectory>
-                    <!-- Github pages only generates documentation from docs
-                        directory -->
-                    <outputDirectory>${basedir}/docs</outputDirectory>
-                </configuration>
-                <dependencies>
-                    <dependency>
-                        <groupId>org.apache.maven.doxia</groupId>
-                        <artifactId>doxia-module-markdown</artifactId>
-                        <version>1.8</version>
-                    </dependency>
-                </dependencies>
-            </plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-site-plugin</artifactId>
+				<version>3.7.1</version>
+				<configuration>
+					<!-- This is where site.xml is located -->
+					<siteDirectory>${basedir}/wiki</siteDirectory>
+					<!-- Github pages only generates documentation from docs directory -->
+					<outputDirectory>${basedir}/docs</outputDirectory>
+				</configuration>
+				<dependencies>
+					<dependency>
+						<groupId>org.apache.maven.doxia</groupId>
+						<artifactId>doxia-module-markdown</artifactId>
+						<version>1.8</version>
+					</dependency>
+				</dependencies>
+			</plugin>
 
+			<plugin>
```

Comment:
This is duplicated code.
And indention is still tabs.

Reply:
done